### DISCUSSION TITLE
MDE seeing "Meetings for Mac" from Zoom in kitty.app

### DISCUSSION THREAD

**Discussion by charliwest** [kovidgoyal/kitty#7064] [Q&A]
Hello all, bit random, but does anyone know why Microsoft Defender would see kitty as being/having "Meetings for Mac" from Zoom in it?
It is being flagged by our cyber sec and not sure why on earth it would think that :)

**Comment by kovidgoyal** [comment]
Probably because kitty contains some compiled objective-C code that
matches whatever heuristic defender uses to identify "Meetings for Mac"

**Reply by charliwest** [reply]
Thanks, we have reported the mistake to MS....maybe in a year they will fix it :D
